mne/viz/topomap.py

```diff
@@ -669,7 +669,7 @@ def plot_topomap(data, pos, vmin=None, vmax=None, cmap=None, sensors=True,
         cont = None  # can't make contours for constant-valued functions
     else:
         cont = ax.contour(Xi, Yi, Zi, contours, colors='k',
-                          linewidths=linewidth)
+                          linewidths=linewidth / 2)
```

Comment:
better as `/ 2.` for Py2k

Reply:
Don't we `from future import division` (or a similar thing in `six`)?

Reply:
No not in this file

https://github.com/cbrnr/mne-python/blob/34ba4de23ae6ff9a143acf8e9b6f64ab5701489b/mne/viz/topomap.py#L2

Reply:
I see. But I guess it's done somewhere before importing this module, so it should work.

Anyway, I'll change it, but I'd rather add the `future` import above. If you insist, I'll put `2.`.

Reply:
> I guess it's done somewhere before importing this module, so it should work.

Nope -- imports (including `__future__`) have local (file) scope only. Creating `untitled2.py` with:
```
from __future__ import division
my_number = 2
```
Then creating `untitled1.py` with:
```
from untitled2 import my_number
print(2 / my_number)
print(1 / my_number)
```
You get `1` then `0`.

Reply:
Thanks for pointing this out! This is a nice example, although I would put it the other way round (import division in main, do the division in the module without the import). Same result though.